tests/phpunit_aliases.php

```diff
@@ -4,7 +4,7 @@
         trigger_error(sprintf('Your PHPUnit Version must be at least 5.7.0 to use CakePHP Testsuite, found %s', \PHPUnit_Runner_Version::id()), E_USER_ERROR);
     }
 
-    if (!interface_exists('PHPUnit_Framework_Test') && class_exists('PHPUnit_Framework_TestCase')) {
+    if (!class_exists('PHPUnit\Runner\Version')) {
```

Comment:
I chose this class check after resolving maze of BC/FC aliases from Codeception [2.3.9](https://github.com/Codeception/Codeception/blob/2.3.9/shim.php#L16-L52) and PHPUnit `ForwardCompatibility` dir at [5.7.14](https://github.com/sebastianbergmann/phpunit/tree/5.7.14/src/ForwardCompatibility) and [5.7.27](https://github.com/sebastianbergmann/phpunit/tree/5.7.27/src/ForwardCompatibility)